### ISSUE TITLE
Road Mark Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#444]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Road Mark Detection</br>
:red_circle: **Aim** : The aim of this project is to detect road marks using DL methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/pkdarabi/road-mark-detection</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Madhusri02** [comment]
Full name : Madhu Sri
GitHub Profile Link : https://github.com/Madhusri02
Email ID : madhusri0774@gmail.com
Participant ID (if applicable): -
Approach for this Project : We can use Convolutional Neural Network (CNN), Mask R-CNN , U-Net , YOLO algorithms to make this model .
What is your participant role? (SWOC) : Contributor

**Comment by abhisheks008** [comment]
Are you a participant of SWOC?

**Comment by abhisheks008** [comment]
Assigned @Madhusri02

**Comment by Madhusri02** [comment]
Thank You @abhisheks008

**Comment by SrinadhVura** [comment]
Hi there,

I would like to work on this issue.
Full name : Srinadh Vura
GitHub Profile Link : https://github.com/SrinadhVura/
Email ID : srinadhvura@gmail.com
Participant ID (if applicable):
Approach for this Project : I have two approaches in my mind. First one is to get a pre-trained resnet-50 without top layers and train it on the dataset. Second one is to apply a CNN that detects edges as animals shape differs to a greater extent from one another
What is your participant role? Contributor | GSSOC 2024

**Comment by abhisheks008** [comment]
Hi @SrinadhVura  wait for the induction session to complete by today evening, after that issues will be assigned to the contributors.

**Comment by SHREERAJ11** [comment]
Full name : Sobhan Shreeraj Sa
    GitHub Profile Link : https://github.com/SHREERAJ11
    Email ID : sobhansa032gmail.com
    Participant ID (if applicable): --
    Approach for this Project : My primary approach would be trying multiple pre-trained CNN models. Apart from CNN, I would also like to try SVM or Decision Trees for comparing accuracy scores with CNNs and I will try to learn and implement Semantic segmentation with FCNs.
    What is your participant role? GSSoC | Contributor

I would really like to work on this issue!

**Comment by abhisheks008** [comment]
Hi @SHREERAJ11 thanks for your interest. As the repository name suggests, the projects are mainly focusing on deep learning methods and you need to implement at least 2-3 deep learning algorithms for any of the projects. Can you please rephrase your approach?

**Comment by Vinayakkh** [comment]
Full name : Vinayak Khandelwal
GitHub Profile Link :https://github.com/Vinayakkh
Email ID :2021pietcsvinayak180@poornima.org
Participant ID (if applicable):
Approach for this Project :Apply CNN to this and by applying layers to make it more accurate prediction on train data applying various models
What is your participant role? GSSOC | Contributor

**Comment by abhisheks008** [comment]
Hi @Vinayakkh can you please share the algorithms/models you are planning to use for this dataset. As you need to implement at least 2-3 models, and compare them based on the accuracy scores to find out the best fitted model.

**Comment by SHREERAJ11** [comment]
Hello @abhisheks008 , sorry for the confusion. Initially I meant to apply image segmentation CNN approaches: Mask R-CNN, U-NET, PSP-NET and DeepLab. Also I would like to try with other algos if permitted to check for accuracies and errors.

**Comment by abhisheks008** [comment]
Nice approach @SHREERAJ11. You can start working on this issue.
